q2_mlab/db/tests/test_mapping.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+
+from q2_mlab.db.mapping import (
+    remap_parameters,
+    trivial_remapper,
+    string_or_number_remapper,
+    serialize_remapper,
+)
+
+
+class ParameterRemapTestCase(unittest.TestCase):
+    def test_trivial_remapper(self):
+        parameter = 'param'
+        value = 24
+        obs = trivial_remapper(parameter, value)
+        self.assertDictEqual({'param': 24}, obs)
+
+    def test_string_or_number_remapper(self):
```

Comment:
Let's add a test for None parameters, or maybe cast those to strings.

`string_or_number_remapper` is giving a `TypeError: 'NoneType' object is not iterable` when the parameter is None, as in this example:
`{'bootstrap': False, 'criterion': 'mae', 'max_depth': 7, 'max_features': None, 'max_samples': 0.75, 'min_samples_leaf': 0.41000000000000003, 'min_samples_split': 2, 'n_estimators': 1000, 'n_jobs': -1, 'random_state': 2020}`